app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,51 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
+    <%= stylesheet_link_tag "application", media: "all", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag 'recipes', 'data-turbolinks-track': 'reload' %>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.1.3/js/bootstrap.bundle.min.js"></script>
   </head>
-
+  
   <body>
+    <% if notice %>
+    <p class="flash notice"><%= notice %></p>
+    <% end %>
+    <nav class="navbar navbar-expand-lg bg-body-tertiary">
+      <div class="container-fluid">
+          <%= image_tag 'tdlogo.png', class: 'logoMenu', alt: 'Taste Diary Logo' %>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <%= link_to 'Home', root_path, class: 'nav-link active' %>
+            </li>
+            <% if user_signed_in? %>
+              <li class="nav-item">
+                <%= link_to "My Recipes", recipes_path, class: "nav-link" %>
+              </li>
+              <li class="nav-item">
+                <%= link_to "Ingredients", ingredients_path, class: "nav-link" %>
+              </li>
+              <li class="nav-item">
+                <%= link_to 'Shopping List', shopping_list_path, class: "nav-link" %>
+              </li>
+            <% end %>
+          </ul>
```

Comment:
- [ ] Please make sure you have a navigation link to public recipes page listing all publicly available recipes from other users with their names and total prices. The page has to match the structure of the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true) and url `/public_recipes`:pray: 
  
  ![image](https://github.com/zamcham/TasteDiary/assets/92303891/c42872b1-f337-41ac-a05c-9d96e1df6ded)